shared/transactions/credential_accept.yaml

```diff
@@ -0,0 +1,36 @@
+components:
+  schemas:
+    CredentialAcceptTransaction:
+      $id: CredentialAcceptTransaction
+      allOf:
+        - $ref: '../base.yaml#/components/schemas/BaseTransaction'
+      type: object
+      description: |
+        A CredentialAccept transaction accepts a credential, which makes the credential valid. Only the subject of the credential can do this.
+      required:
+        - CredentialType
+        - Issuer
+      properties:
+        CredentialType:
```

Comment:
We should add a custom rule for valid credential type. Same applies to other transactions